src/etc/one-context.d/loc-14-mount-swap##one

```diff
@@ -21,6 +21,10 @@ if [ "$1" != 'local' ] ; then
     exit 0
 fi
 
+if [ "${DISABLE_SWAP}" == "true" ]; then
```

Comment:
We use `YES`, `yes`,  `NO`, `no` as the boolean values in the context.

Reply:
The context parameter `DISABLE_SWAP` is misleading - swap can still be enabled if there is correct line in `/etc/fstab` for example - this script is looking for block devices with a type `SWAP` and mounts them (if on linux) - I think that a better name but still short is `IGNORE_SWAP`.